test/functionalTest/cases/2193_entity_not_found_when_forwarding/entity_not_found_when_forwarding_specific_attr.test

```diff
@@ -0,0 +1,104 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Entity not found in lying context provider, specific attr registered
+
+--SHELL-INIT--
+dbInit CB
+dbInit CP1
+brokerStart CB
+brokerStart CP1
+
+--SHELL--
+
+#
+# 01. Register E1/T1/A1 in CB, provApp: CP1
+# 02. Modify E1/A1 in CP1 via CB, using PUT /v2/entities/E1/attrs/A1?type=T
+#
+
+echo "01. Register E1/T1/A1 in CB, provApp: CP1"
+echo "========================================="
+payload='{
+  "contextRegistrations": [
+  {
+    "entities": [
+      {
+         "type": "T1",
+         "isPattern": "false",
+         "id": "E1"
+      }
+    ],
+    "attributes": [
+      {
+        "name": "A1",
+        "type": "string",
+        "isDomain": "false"
+      }
+    ],
+    "providingApplication": "http://localhost:'${CP1_PORT}'/v1"
+    }
+ ],
+ "duration": "P1M"
+}'
+orionCurl --url /v1/registry/registerContext --payload "$payload"
+echo
+echo
+
+
+echo "02. Modify E1/A1 in CP1 via CB, using PUT /v2/entities/E1/attrs/A1?type=T1, see error"
+echo "====================================================================================="
+payload='{ "value": "05" }'
+orionCurl --url '/v2/entities/E1/attrs/A1?type=T1' --payload "$payload" -X PUT
+echo
+echo
+
+
+--REGEXPECT--
+01. Register E1/T1/A1 in CB, provApp: CP1
+=========================================
+HTTP/1.1 200 OK
+Content-Length: 74
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "duration": "P1M",
+    "registrationId": "REGEX([0-9a-f]{24})"
+}
+
+
+02. Modify E1/A1 in CP1 via CB, using PUT /v2/entities/E1/attrs/A1?type=T1, see error
+=====================================================================================
+HTTP/1.1 404 Not Found
+Content-Length: 0
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
```

Comment:
Note this message hasn't payload (and it should). Probably this is due to the "Fix: missing error payload for NotFound error in 'PUT /v2/entities/<eId>/attrs/<attrName>' and 'PUT /v2/entities/<eId>/attrs/<attrName>/value' operations" has not reach this PR.